ew/cmd/item/itemcmds.py

```diff
@@ -407,9 +428,10 @@ async def inventory_print(cmd):
 
             if not stacking:
                 # Generate the item's line in the response based on the specified formatting
-                response_part = "\n{id_item}: {soulbound_style}{name}{soulbound_style}{quantity}".format(
+                response_part = "\n{id_item}: {soulbound_style}{hue}{name}{soulbound_style}{quantity}".format(
```

Comment:
i saw you pulled rigor mortis'd items earlier but they don't seem to change the item's display. am i missing something?

Reply:
It filters the inventory down to only rigor mortis'd items instead of displaying preserved infront of the name. I could have both but its sort of redundant.